CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+## How to Contribute
+
+We welcome contributions in the form of issues or pull requests!
```

Comment:
for extra awesomeness, you should make `issue` and `pull requests` hyperlinks to the list of open issues and pull requests, respectively.

Reply:
I'd rather not in case the repo location changes.